hpOneView/oneview_client.py

```diff
@@ -103,7 +103,9 @@
 from hpOneView.resources.uncategorized.os_deployment_plans import OsDeploymentPlans
 from hpOneView.resources.uncategorized.os_deployment_servers import OsDeploymentServers
 from hpOneView.resources.security.certificate_rabbitmq import CertificateRabbitMQ
+from hpOneView.resources.security.login_details import Login_Details
```

Comment:
Probably align with @soodpr for one of you to be sending the `LoginDetails` and another to be sending `VersionDetails`.